pyproject.toml

```diff
@@ -43,6 +43,7 @@ test = [
   "pytest",
   "pytest-cov",
   "pytest-django",
+  "model-bakery",
```

Comment:
On second thought, I'd better if you add a second test run without the dependency. It's good to test that it works without it.